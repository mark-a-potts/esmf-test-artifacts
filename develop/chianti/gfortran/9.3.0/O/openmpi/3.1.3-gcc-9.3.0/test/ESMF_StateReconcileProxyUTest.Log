build time -- 2021-04-23 11:58:54.445068
20210423 120556.033 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120556.033 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120556.033 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120556.033 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120556.033 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120556.033 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120556.033 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120556.033 INFO             PET0 ESMF_COMM                   : openmpi
20210423 120556.033 INFO             PET0 ESMF_MOAB                   : enabled
20210423 120556.033 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 120556.033 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 120556.033 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 120556.033 INFO             PET0 ESMF_PIO                    : enabled
20210423 120556.033 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 120556.033 INFO             PET0  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
20210423 120556.033 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210423 120556.037 INFO             PET0  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
20210423 120556.037 INFO             PET0  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
20210423 120556.037 INFO             PET0  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
20210423 120556.037 INFO             PET0  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
20210423 120556.039 INFO             PET0  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
20210423 120556.039 INFO             PET0  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
20210423 120556.039 INFO             PET0  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
20210423 120556.039 INFO             PET0  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
20210423 120556.039 INFO             PET0 Finalizing ESMF
20210423 120556.033 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120556.033 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120556.033 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120556.033 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120556.033 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120556.033 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120556.033 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120556.033 INFO             PET1 ESMF_COMM                   : openmpi
20210423 120556.033 INFO             PET1 ESMF_MOAB                   : enabled
20210423 120556.033 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 120556.033 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 120556.033 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 120556.033 INFO             PET1 ESMF_PIO                    : enabled
20210423 120556.033 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 120556.033 INFO             PET1  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
20210423 120556.033 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210423 120556.037 INFO             PET1  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
20210423 120556.037 INFO             PET1  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
20210423 120556.037 INFO             PET1  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
20210423 120556.037 INFO             PET1  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
20210423 120556.039 INFO             PET1  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
20210423 120556.039 INFO             PET1  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
20210423 120556.039 INFO             PET1  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
20210423 120556.039 INFO             PET1  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
20210423 120556.039 INFO             PET1 Finalizing ESMF
20210423 120556.033 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120556.033 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120556.033 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120556.033 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120556.033 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120556.033 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120556.033 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120556.033 INFO             PET2 ESMF_COMM                   : openmpi
20210423 120556.033 INFO             PET2 ESMF_MOAB                   : enabled
20210423 120556.033 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 120556.033 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 120556.033 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 120556.033 INFO             PET2 ESMF_PIO                    : enabled
20210423 120556.033 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 120556.033 INFO             PET2  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
20210423 120556.033 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210423 120556.037 INFO             PET2  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
20210423 120556.037 INFO             PET2  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
20210423 120556.037 INFO             PET2  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
20210423 120556.037 INFO             PET2  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
20210423 120556.039 INFO             PET2  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
20210423 120556.039 INFO             PET2  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
20210423 120556.039 INFO             PET2  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
20210423 120556.039 INFO             PET2  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
20210423 120556.039 INFO             PET2 Finalizing ESMF
20210423 120556.033 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120556.033 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120556.033 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120556.033 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120556.033 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120556.033 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120556.033 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120556.033 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120556.033 INFO             PET3 ESMF_COMM                   : openmpi
20210423 120556.033 INFO             PET3 ESMF_MOAB                   : enabled
20210423 120556.033 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 120556.033 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 120556.033 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 120556.033 INFO             PET3 ESMF_PIO                    : enabled
20210423 120556.033 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 120556.033 INFO             PET3  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
20210423 120556.033 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210423 120556.037 INFO             PET3  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
20210423 120556.037 INFO             PET3  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
20210423 120556.037 INFO             PET3  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
20210423 120556.037 INFO             PET3  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
20210423 120556.039 INFO             PET3  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
20210423 120556.039 INFO             PET3  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
20210423 120556.039 INFO             PET3  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
20210423 120556.039 INFO             PET3  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
20210423 120556.039 INFO             PET3 Finalizing ESMF
