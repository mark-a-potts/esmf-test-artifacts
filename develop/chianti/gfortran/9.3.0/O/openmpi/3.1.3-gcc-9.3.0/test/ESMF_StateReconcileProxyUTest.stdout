build time -- 2021-04-23 11:58:54.445068
  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_StateReconcileProxyUTest.F90, line 244
  NUMBER_OF_PROCESSORS           4
  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
  PASS  Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 283
  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
  PASS  Ensure g1 and g2 are aliases to the same Grid (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 319
  PASS  Ensure m1 and m2 are aliases to the same Mesh (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 336
  PASS  Ensure l1 and l2 are aliases to the same LocStream (proxy) Test, ESMF_StateReconcileProxyUTest.F90, line 353
  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
  PASS  Re-Reconciling a State, ESMF_StateReconcileProxyUTest.F90, line 363
  PASS  Ensure Re-Reconcile Field (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 372
  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
  PASS  Ensure Re-Reconcile FieldBundle (proxy) persistence Test, ESMF_StateReconcileProxyUTest.F90, line 381
  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
  Ending Test, file ESMF_StateReconcileProxyUTest.F90, line 400
  PET           1  Test Elapsed Time    7.0929999999999982       msec.
  PET           2  Test Elapsed Time    6.9489999999999990       msec.
  PET           3  Test Elapsed Time    6.9979999999999993       msec.
  PET           0  Test Elapsed Time    7.1810000000000000       msec.
